docs/uptime/monitoring/install-icinga2-monitoring-on-debian-9.md

```diff
@@ -19,15 +19,15 @@ external_resources:
  - '[Official Icinga Documentation](https://www.icinga.com/docs/icinga2/latest/doc/01-about/)'
 ---
 
-![Install Icinga2 Monitoring on Debian 9](/docs/assets/icinga/Icinga2.jpg "Install Icinga2 Monitoring on Debian 9")
+![Install Icinga 2 Monitoring on Debian 9](/docs/assets/icinga/Icinga 2.jpg "Install Icinga 2 Monitoring on Debian 9")
```

Comment:
This will break the image path

Reply:
Ouch, and I thought I had carefully reviewed what sed gave me. I guess I was wrong. :)

Reply:
lgtm, will merge on green

Reply:
Cool, thanks. 😊